packages/openactive-broker-microservice/src/util/criteria-oriented-opportunity-id-cache.js

```diff
@@ -76,6 +82,61 @@ const CriteriaOrientedOpportunityIdCache = {
     if (!typeBucket) throw new Error(`The specified opportunityType (${opportunityType}) is not currently supported.`);
     return typeBucket;
   },
+
+  /**
+   * ! `cache` is mutated.
+   *
+   * @param {OpportunityIdCacheType} cache
+   * @param {string} opportunityId
+   * @param {object} args
+   * @param {string} args.criteriaName
+   * @param {string} args.bookingFlow
+   * @param {string} args.opportunityType
+   * @param {string} args.sellerId
+   */
+  setOpportunityMatchesCriteria(cache, opportunityId, { criteriaName, bookingFlow, opportunityType, sellerId }) {
```

Comment:
This makes some of the code clearer. But I did this so that I could use these functions to set up tests